### DISCUSSION TITLE
Start action on QR Code detection

### DISCUSSION THREAD

**Discussion by dimitribeyondgravity** [gopro/labs#473] [Q&A] [answered]
Hi !
Is there a way to start an action when the camera detects a QR Code?

I know it's possible to juste to load an existing script saved on the camera, with "+Lscriptname" command, but here i would like to avoid modifying the QRCode value by adding this command.

So basically it would be a script that include a conditional start, based on qr code detection statut.

Does anybody knows how to do that?

Thank you a lot !

**Comment by dnewman-gpsw** [comment, marked as answer]
!Lscript would be the easy solution, I don't understand why that would not work.  The problem any valid QR Code will replace the script currently running.  So script look for QR code, is stopped upon a new QR code being presented.  This is by design, so a Labs camera can be de-bricked without pulling the battery (if a bad script is created.)

**Reply by dimitribeyondgravity** [reply]
Very interesting to know it works that way, it's going to help ! Thank you!
